core/kazoo_convert/src/kz_fax_converter.erl

```diff
@@ -370,8 +384,9 @@ format_output(FilePath, #{<<"output_type">> := 'binary'}) ->
     end;
 format_output(FilePath, #{<<"output_type">> := 'path'}) ->
     {'ok', FilePath};
-format_output(FilePath, _Options) ->
-    {'ok', FilePath}.
+format_output(FilePath, Options) ->
+    read_file({'file', FilePath}, Options).
```

Comment:
This changes the default format from 'path' to 'binary', whats the need for this? You can always ask for binary if you need binary.

Reply:
Current master implementations is always return `path` and ignores `binary` options.
I missed default value. Will update to
```
format_output(FilePath, _) ->
    {'ok', FilePath};
format_output(FilePath, #{<<"output_type">> := 'binary'}=Options)
    read_file({'file', FilePath}, Options).
```

Reply:
yes, i revert back this my change